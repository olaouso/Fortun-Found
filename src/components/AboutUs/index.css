.aboutUsComponent {
  display: flex;
  color: #590995;
}
.aboutCenter {
  text-align: center;
  margin: 1em 5em;
  font-family: "Acme";
}

.aboutHeader {
  color: #03c4a1;
  margin: 1.5em 0;
  font-size: 3em;
}
.aboutTeamImg {
  width: 60%;
  margin: 2em 0;
}
.teamImgDiv {
  width: 100%;
}
