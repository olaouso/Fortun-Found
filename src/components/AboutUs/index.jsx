import React from "react";
import "./index.css";

import { Row } from "react-bootstrap";
import AboutCard from "../aboutUsCard";
import { useTranslation } from "react-i18next";

export default function AboutUs() {
  const { t } = useTranslation();
  const aboutCardInfo = [...t("about.aboutUs", { returnObjects: true })];

  return (
    <div className="aboutCenter">
      <div>
        <h2 className="aboutHeader">{t("about.about")}</h2>
      </div>
      <Row>
        {aboutCardInfo.map((aboutCard, index) => (
          <AboutCard {...aboutCard} key={index} />
        ))}
      </Row>
    </div>
  );
}
