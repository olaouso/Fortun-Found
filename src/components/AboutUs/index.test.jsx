import React from "react";
import renderer from "react-test-renderer";
import AboutUs from "./";

it("renders AboutUs correctly", () => {
  const tree = renderer.create(<AboutUs />).toJSON();
  expect(tree).toMatchSnapshot();
});
