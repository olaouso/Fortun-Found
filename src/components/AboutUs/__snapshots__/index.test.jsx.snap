// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AboutUs correctly 1`] = `
<div
  className="aboutCenter"
>
  <div>
    <h2
      className="aboutHeader"
    >
      about.about
    </h2>
  </div>
  <div
    className="row"
  >
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
    <div
      className="col-md-4"
    >
      <h4
        className="cardTitle"
      />
      <p
        className="cardParagraph"
      />
    </div>
  </div>
</div>
`;
